physics.py

```diff
@@ -9,7 +9,10 @@
 # of your robot code without too much extra effort.
 #
 
+import functools
+import operator
```

Comment:
Are these in robotpy? I assume so but want to ask to make sure it'll work on stuff

Reply:
default python